di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -77,7 +78,13 @@ public class IpvHandler {
                 String errorType = request.queryMap().get("error").value();
                 String userIdTextValue = request.queryMap().get("userIdText").value();
                 String signInJourneyIdText = request.queryMap().get("signInJourneyIdText").value();
-                String[] vtr = request.queryMap().get("vtrText").values();
+                QueryParamsMap queryParamsMap = request.queryMap().get("vtrText");
+                String[] vtr;
+                if (queryParamsMap != null) {
+                    vtr = queryParamsMap.values();
+                } else {
+                    vtr = new String[] {};
```

Comment:
The problem Chris highlighted is that when the 'error' journey form is submitted, there is no VTR.

Your change will make it so that this case will return a default of empty VTR, but I think this isn't very representative. I wonder if we should make the default value `Cl.Cm.P2` as with the main form.

I think you could either:
- Set the default here, if the `vtrText` is not specified
- Add a hidden input to the error form, with the default